src/raftstore/store/fsm/apply.rs

```diff
@@ -2288,6 +2290,8 @@ pub enum Msg {
     Destroy(Destroy),
     Snapshot(GenSnapTask),
     #[cfg(test)]
+    Testmsg(Sender<i32>),
```

Comment:
Why not use `Validate`?

Reply:
Yes，I could use Validate。I didn't notice it when I coded, just wrote it according to my own thoughts.